src/app/components/List/EBookList.jsx

```diff
@@ -25,16 +26,24 @@ const generateLink = (url, eReaderUrl) => {
   );
 };
 
-const EBookList = (props) => {
-  return (
-    <ul className="nypl-items-list">
-      {props.ebooks.map((ebook, ebookKey) => (
-        <li key={ebookKey.toString()}>
-          {generateLink(ebook.url, props.eReaderUrl)} {(ebook.epub_path) ? [ebook.epub_path.split('/').pop()] : ''}
-        </li>
-      ))}
-    </ul>
-  );
+const EBookList = ({ ebooks, eReaderUrl }) => (
+  <ul className="nypl-items-list">
+    {ebooks.map((item, ebookKey) => item.links.map((link, linkKey) => (
```

Comment:
Do you not need to differentiate all the links in each item? If you do, then this should be a new `ul` since it's a list of lists. Otherwise, having a comment about how the data should be displayed will be helpful.

Reply:
this design is going to change anyway